data-raw/simulating-DAG.R

```diff
@@ -28,18 +85,50 @@ simulated_data <- simulateSEM(graph, N = 2500) %>%
     setNames(paste0("metabolite_", 1:length(.)))
 
 devtools::use_data(simulated_data, overwrite = TRUE)
+>>>>>>> 6d0f48698be38b8c1d900db5c10e53eee6b09a3d
 
 
-X_DAG<-est.pcor.skel.DAG.adj(dat=x,alpha_val=0.05)
-plot(X_DAG$skel_est)
+    # calculate censoring variable
+    cens = matrix(0,n,)
+    for(i in 1:n){
+        if (fup_time[i] < p10){cens[i]<-1}
+        else {fup_time[i] <-p10}
+    }
+    cens<-as.numeric(cens)
+    #print(mean(fup_time))
+    print(max(fup_time))
```

Comment:
Shouldn't have side-effects (aka using the `print` function in the middle of a function)